src/consensus/consensus.h

```diff
@@ -22,6 +22,8 @@ static const unsigned int MAX_BLOCK_SIGOPS = 20000;
 static const unsigned int MAX_TX_SIZE = 100000;
 /** Coinbase transaction outputs can only be spent after this number of new blocks (network rule) */
 static const int COINBASE_MATURITY = 100;
+/** The minimum value which is invalid for expiry height, used by CTransaction and CMutableTransaction */
```

Comment:
Non-blocking but I had to read this a few times to grok.  Maybe something like this is easier to read: " A valid expiry height must be lower than this threshold value, used by ..."